File: Find Day/Find_Day.java
import java.util.*;
class Find_Day{
    int day(int d,int m,int y){
        int yc=0,day=0;
        int mc[]={0,3,3,6,1,4,6,2,5,0,3,5};
        if(y>=1600 && y<=1699)
            yc=6;
        else if(y>=1700 && y<=1799)
            yc=4;
        else if(y>=1800 && y<=1899)
            yc=2;
        else if(y>=1900 && y<=1999)
            yc=0;
        else if(y>=2000 && y<=2099)
            yc=6;
        
        day=d+mc[m-1]+(y%100)+((y%100)/4)+yc;
        day=day%7;
        return day;
    }

    public static void main(String[] args){
        Find_Day ob=new Find_Day();
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter in Number");
        System.out.print("Enter Date : ");
        int d=sc.nextInt();
        System.out.print("Enter Month : ");
        int m=sc.nextInt();
        System.out.print("Enter Year : ");
        int y=sc.nextInt();
        sc.close();
        System.out.println("Date : "+d+"-"+m+"-"+y);
        int days=ob.day(d,m,y);
        String da="";
        switch(days){
            case 0:
                da="Sunday";
            break;
            case 1:
                da="Monday";
            break;
            case 2:
                da="Tuesday";
            break;
            case 3:
                da="Wednesday";
            break;
            case 4:
                da="Thursday";
            break;
            case 5:
                da="Friday";
            break;
            case 6:
                da="Saturday";
            break;
        }
        System.out.print("The Day is : "+da);


    }
}

File: Problem Statements/Find Day/Find_Day.py
def day(d,m,y):
    yc,day=0,0
    mc=(0,3,3,6,1,4,6,2,5,0,3,5)
    if y>=1600 and y<=1699:
        yc=6
    elif y>=1700 and y<=1799:
        yc=4
    elif y>=1800 and y<=1899:
        yc=2
    elif y>=1900 and y<=1999:
        yc=0
    elif y>=2000 and y<=2099:
        yc=6
    
    day=day+mc[m-1]+(y%100)+((y%100)//4)+yc
    day=day%7
    return day

print("Enter in Numbers")
d=int(input("Enter Date : "))
m=int(input("Enter Month : "))
y=int(input("Enter Year : "))
print("The Date is : {}-{}-{}".format(d,m,y))
days=day(d,m,y)
if days==0:
    s="Sunday"
elif days==1:
    s="Monday"
elif days==2:
    s="Tuesday"
elif days==3:
    s="Wednesday"
elif days==4:
    s="Thursday"
elif days==5:
    s="Friday"
elif days==6:
    s="Saturday"
print("The Day is : {}".format(s))